src/Metadata/Metadata.php

```diff
@@ -225,25 +225,6 @@ public function withFilters(array $filters): static
         return $self;
     }
 
-    /**
-     * @deprecated this will be removed in v4
```

Comment:
I didn't delete the attribute in the constructor to avoid changing the order of the parameters. Tell me if we can remove it

Reply:
I finally deleted them to fix the tests. In any case, it must be used with the named parameters and avoids this problem.